### ISSUE TITLE
Title/ Loading Screens

### ISSUE THREAD

**Issue by violetskyzz** [SupremeObsidian/ProjectManager#83]

**Comment by dryter97** [comment]
Hem.... If you want something way more custom its working with optifine
else not its only the mojang sprite and they can only be saw once the
ressource pack is loaded wich is... Maybe useless. About the optifine one
it could be interesting. Other thing i could suggest is making a font pack
in to add caracther sprite

**Comment by violetskyzz** [comment]
Yes the plan is to make the pack available on the website for players to download client side, in which case they would be able to see the title screen. Though its not a super important aspect, I feel like to have a well rounded texture pack all of the elements should be included.
